code/datums/spells/alien_spells/basetype_alien_touch.dm

```diff
@@ -0,0 +1,56 @@
+/obj/effect/proc_holder/spell/touch/alien_spell
+	name = "Basetype Alien spell"
+	desc = "You should not see this in game, if you do file a github report!"
+	hand_path = "/obj/item/melee/touch_attack/alien"
+	/// Extremely fast cooldown, only present so the cooldown system doesn't explode
+	base_cooldown = 1
+	action_background_icon_state = "bg_alien"
+	clothes_req = FALSE
+	create_attack_logs = FALSE
+	/// Every alien spell creates only logs, no attack messages on someone placing weeds, but you DO get attack messages on neurotoxin and corrosive acid
+	create_only_logs = TRUE
+	/// We want a special on remove message, so we got this variable to do so
+	on_remove_message = FALSE
+	/// How much plasma it costs to use this
+	var/plasma_cost = 0
+	action_icon_state = "gib"
+
+/obj/effect/proc_holder/spell/touch/alien_spell/Initialize(mapload)
+	. = ..()
+	if(plasma_cost)
+		name = "[name] ([plasma_cost])"
+
+/obj/effect/proc_holder/spell/touch/alien_spell/Click(mob/user = usr)
+	if(attached_hand)
+		to_chat(user, "<span class='noticealien'>You withdraw your [src].</span>")
```

Comment:
An `on_unprepare` or so, would be cleaner instead of overriding `Click`. Overriding `Click` was still something I wanted to refactor away from the `touch` subtype.